### ISSUE TITLE
Gitpod for Startups: Add to pricing page & Footer

### ISSUE THREAD

**Issue by phimae** [gitpod-io/website#1093]
Hi @nisarhassan12,

As discussed, there are 3 things that need to be done before the launch and merged on October 25 latest.

- [x]  Add a "Gitpod for Startups" section in the pricing page, as per the below mockup - see comments in this thread for link and bold sections (sorry, I don't have it in Figma)

![image](https://user-images.githubusercontent.com/83109781/138039377-23b70c1b-f222-4fb7-93c6-7150b1dcb8fa.png)

Here is the text:

> Our startup program gives [eligible](https://www.gitpod.io/for/startups) startups up to **2 years of Gitpod for free**, no matter how fast your developers and workloads scale. Spend less time worrying about brittle, local dev environments and more time on building great things.

- [x]  remove the comment for "Gitpod for startups" in the footnote as discusses [here](https://github.com/gitpod-io/website/pull/1044#:~:text=Thanks%20for%20including%20it%20-%20can%20you%20move%20it%20to%20%22Gitpod%22%2C%20after%20Self-hosted%20(sorry%2C%20my%20bad)%20and%20comment%20it%20out%20for%20now%3F%20(Reason%3A%20We%20will%20stealth%20launch%20the%20startups%20page%20so%20that%20you%20can%20access%20it%20via%20link%2C%20but%20shouldn%27t%20find%20anything%20on%20the%20main%20page%20until%20the%20official%20launch))
- [ ] #1095

Let me know if you have any questions, thanks!

**Comment by nisarhassan12** [comment]
This can be closed as the remaining todo has its own separate issue.
